addon/plugins/variable-plugin/nodes.ts

```diff
@@ -13,7 +13,9 @@ import { v4 as uuidv4 } from 'uuid';
 const CONTENT_SELECTOR = `span[property~='${EXT('content').prefixed}'], 
                           span[property~='${EXT('content').full}']`;
 
-const emberNodeConfig: EmberNodeConfig = {
+// Need to update EmberNodeConfig to include the needsFFKludge property there
+// Or allow to augment it somehow else?
+const emberNodeConfig: EmberNodeConfig & { needsFFKludge: boolean } = {
```

Comment:
the `createEmberNodespec` function already passes through extra keys, so we just need to fix the type of EmberNodeConfig by adding:
```ts
  [key: string]: unknown;
```